src/views/product-root-docs-path-landing/components/hero/index.tsx

```diff
@@ -19,6 +19,7 @@ const ProductRootDocsPathLandingHero = ({
           productSlug={
             currentProduct.slug === 'sentinel' ? 'hcp' : currentProduct.slug
           }
+          className={s.icon}
```

Comment:
🙋🏻 Do you know if it's possible to leverage our `g-hide-on-mobile` global helper here?

Reply:
That doesn't seem to work because the [icon tile `root` class ](https://github.com/hashicorp/dev-portal/blob/main/src/components/icon-tile/icon-tile.module.css#L6)takes precedence over it.